docs/sources/reference/components/pyroscope.scrape.md

```diff
@@ -65,6 +65,7 @@ Name                | Type                     | Description
 `params`            | `map(list(string))`      | A set of query parameters with which the target is scraped.        |                | no
 `scrape_interval`   | `duration`               | How frequently to scrape the targets of this scrape configuration. | `"15s"`        | no
 `scrape_timeout`    | `duration`               | The timeout for scraping targets of this configuration. Must be larger than `scrape_interval`. | `"18s"` | no
+`profiling_duration`| `duration` | The duration for a delta profiling to be scraped. Must be larger than 1 second. | `"14s"` | no
```

Comment:
Sorry, one last thing. Could we name this with a little more descriptive name?

```suggestion
`delta_profiling_duration`| `duration` | The duration for a delta profiling to be scraped. Must be larger than 1 second. | `"14s"` | no
```

Reply:
I've changed this in HEAD. PTAL